### PR TITLE
Modal feature

### PR DESCRIPTION
**Pull request by kinginthenorthcodez**
## Modal feature
- Add Dynamic content for project sections
- Add Dynamic Modal for projects

warm regards
@aimalamiri 
@kinginthenorthcodez

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -459,9 +489,154 @@ footer .email {
...
+}
+
+.modal-list {
+  display: flex;
+  list-style: none;
+  width: 245px;
+  height: 48px;
+  margin: 0  auto;
+  align-items: center;
+}
```
Comment (omar-muhamad) [inline assets/css/style.css]: - [x] Kindly add the correct margin value between the project description and the list of technologies like The Figma design.

- [x] I can see the list of technologies overflowing from the popup container and make your site side-scrolling. Please fix that and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | chrome-capture-2022-3-7  |

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi Team,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: assets/css/style.css (hunk 1 above)
Comment (kinginthenorthcodez) [reply assets/css/style.css]: - Fixed margin value between the project description and the list of technologies 👍🏽 
- Fixed list of technologies overflowing from the popup container and  site side-scrolling 👍🏽

**Review by zainsadaqat — APPROVED** [review]
# Hello @kinginthenorthcodez 😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kinginthenorthcodez** [comment]
Merge #10 into main

**Pull request merged**